client/src/controllers/ActionController.test.js

```diff
@@ -169,4 +169,28 @@ describe('ActionController', () => {
       expect(window.location.assign).not.toHaveBeenCalled();
     });
   });
+
+  describe('select method', () => {
+    beforeEach(() => {
+      document.body.innerHTML = `
+        <textarea 
+          id="text" 
+          rows="1" 
+          data-controller="w-action" 
+          data-action="focus->w-action#select">
+          value="some random text"
+        </textarea>
+      `;
+      app = Application.start();
+      app.register('w-action', ActionController);
+    });
+
+    it('select should be called when you click on text in textarea', () => {
+      const textarea = document.getElementById('text');
```

Comment:
```suggestion
      // check that there is no selection initially
            expect(textarea.selectionEnd).toBe(??);
      const textarea = document.getElementById('text');
```

It's always good to have a precondition for you test, confirming that the initial state is 'default'. Then you do something (focus on the element) and then the final state is another expectation (content is selected). 

This way you can be sure that the code is working properly. 

Great effort here, well done on working through the Jest testing stuff.